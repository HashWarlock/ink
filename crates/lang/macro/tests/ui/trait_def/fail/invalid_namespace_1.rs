use ink_lang as ink;

#[ink::trait_definition(namespace = "::invalid::rust::identifier")]
pub trait InvalidNamespace {
    #[ink(message)]
    fn message(&self);
}

fn main() {}
