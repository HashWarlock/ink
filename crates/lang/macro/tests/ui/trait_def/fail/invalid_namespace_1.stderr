error: encountered invalid Rust identifier for the ink! namespace configuration parameter
 --> $DIR/invalid_namespace_1.rs:3:37
  |
3 | #[ink::trait_definition(namespace = "::invalid::rust::identifier")]
  |                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
